### DISCUSSION TITLE
Query in Associate public namespace

### DISCUSSION THREAD

**Discussion by MadanaGopalD** [apolloconfig/apollo#5241] [Q&A]
I have created an public namespace and added few configuration to it. Now I am trying to associate it to another application. After associating  it is asking me to override the configuration. Is it like only after overriding configuration I can access it or how does it work?
Thank you in advance for the reply.
![Screenshot from 2024-09-26 12-06-13](https://github.com/user-attachments/assets/d9186a0e-7a42-44ac-ad07-a7c2b1ebc19b)

**Comment by nobodyiam** [comment]
You don't need to associate a public namespace to get its configurations. You only need to do it when you want to override some configurations.
https://www.apolloconfig.com/#/en/design/apollo-core-concept-namespace
